src/frontends/tensorflow/src/tf_utils.cpp

```diff
@@ -463,12 +464,24 @@ bool propagate_conditional_flow(const OutputVector& ov_inputs,
     return to_propagate;
 }
 
+namespace {
+shared_ptr<op::v0::Parameter> get_parent_parameter(const shared_ptr<op::v0::Result>& node) {
+    const auto input_values = node->input_values();
+    if (input_values.empty())
+        return {};
```

Comment:
Result node should always have one input value specification. I don't think we need to check a number of inputs

Reply:
may be it make sense to check that node is not `nullptr`